### DISCUSSION TITLE
Awareness around CLOMonitor/CLOTributor

### DISCUSSION THREAD

**Discussion by aliok** [cncf/clomonitor#1161] [General]
How does CLOMonitor team make maintainers remember to add their repositories to CLOMonitor?

I believe CLOMonitor and CLOTributor are very nice tools and I am curious how to make CNCF communities have more awareness around them.

**Comment by tegioz** [comment]
At the moment we take care of this ourselves with periodic syncs every now and then (<https://github.com/cncf/clomonitor/pulls?q=is%3Apr+sync+projects+is%3Aclosed+>). But the end goal is to feed from the landscape 🙂

**Reply by aliok** [reply]
Ok, got it.

Just found out https://github.com/cncf/clomonitor/issues/471 Let's close this discussion and move over to that ticket.

This is hard to solve I think :)

**Reply by tegioz** [reply]
We are doing some work on the landscape as well, so hopefully we'll be able to make some progress on that ticket 😇
